doc/fgpu.design.rst

```diff
@@ -177,51 +177,129 @@ slow and require multiple passes over the memory, because
+less efficient than it needs to be. The technique detailed below uses the decomposition as provided by
+Cooley-Tukey to break down a larger transform into smaller 'sub-transforms' where the number of 'sub-transforms'
+is intentionally kept small for efficiency reasons and later combined (same process as the FFT) to form
+the larger transform size. This is a multi-step process and requires some extra notation and math tricks.
 
 Real-to-complex transform
 ~~~~~~~~~~~~~~~~~~~~~~~~~
-Let's introduce some notation to see how this works. Let :math:`n` be the
-number of channels, :math:`x_i` be the (real) time-domain samples and
-:math:`X_k` be the Fourier transform (we divide by :math:`2n` rather than
-:math:`n` because that's the number of input samples used).
+Now for some notation to see how this works. Let :math:`z` be the array of samples (:math:`x_{i}`) of
+length :math:`N`. For clarity, array :math:`x` has length :math:`2N` but is divided into even- and odd- samples
+each of length :math:`N`. In other words, each adjacent pair of real values in :math:`x` is interpreted as the real
+and imaginary components of a complex value.
+
+Let :math:`u_i = x_{2i}` and :math:`v_i = x_{2i+1}`, and let :math:`z_i = u_i + jv_i = x_{2i} + j x_{2i+1}`.
+We will start by computing the Fourier transform of :math:`z`. This is convenient to do because :math:`z` has the
+identical memory layout to :math:`x`, so we can obtain it just by viewing the same memory as containing N complex
+numbers instead of 2N real numbers. In other words, each adjacent pair of real values in :math:`x` is interpreted
+as the real and imaginary components of a complex value.
+
+Let :math:`U`, :math:`V` and :math:`Z` denote the Fourier transforms of :math:`u`, :math:`v` and :math:`z` respectively.
+Since the Fourier transform is a linear operator and we defined :math:`z = u + jv`, we also have :math:`Z = U + jV`.
+
+It is important to remember that both :math:`u` and :math:`v` are real-valued, so :math:`U`
+and :math:`V` are Hermitian symmetric. By re-arranging things we can reconstruct :math:`U` and
+:math:`V` from :math:`Z` using Hermitian symmetry properties. Let :math:`U'`
+be :math:`U` with reversed indices i.e., :math:`U'_k = U_{-k}` where indices are taken
+modulo :math:`n`.
+
+Hermitian symmetry means that :math:`U'_k = U_{-k} = \overline{U_k}` where
+the 'overline' in :math:`\overline{U_k}` denotes conjugation. This is effectively saying that by taking the
+reverse indices in :math:`U_k` we get a conjugated result.
+
+Why is this so? Going back to the original definition for the DFT we saw the complex exponential
+:math:`e^{\frac{-2\pi j}{2N}\cdot ik}` has a variable :math:`k` where :math:`k` represents the
+frequency component under computation for the input sequence :math:`x_i`. If :math:`k` is reversed
+(i.e. negative) the complex exponential changes to :math:`e^{\frac{2\pi j}{2n}\cdot i(k)}` as the negaitive
+in :math:`-k` multiplies out.
```

Comment:
@james-smith-za do you have an opinion on whether this proof of a standard Fourier result in Fourier is adding anything? I feel like it interrupts the flow of the discussion.

Reply:
I see your point, but I'm not a person who remembers the finer points of mathematical proofs from undergraduate days very well, so I feel as though it does add some value.

Does Sphinx do footnotes? That may be a good compromise.